src/sst/elements/memHierarchy/tests/testsuite_default_memHierarchy_memHSieve.py

```diff
@@ -34,7 +34,6 @@ def test_memHSieve(self):
 
     def memHSieve_Template(self, testcase, testtimeout=360):
 
-        pin2defined = testing_is_PIN2_used()
```

Comment:
The else branch for a `if pin3defined:` conditional later on states code exists for Pin 2, but it isn't obvious to me what it does, so I didn't remove it.  If someone (@gvoskuilen ?) could tell me if it's safe to remove, that would be good.